includes/Admin/Admin_Page.php

```diff
@@ -307,7 +315,13 @@ public function filter_plugin_action_links( $actions, $plugin_file, $plugin_data
 			return $actions;
 		}
 
-		if ( current_user_can( 'activate_plugins' ) ) {
+		if ( is_multisite() && current_user_can( 'manage_network_plugins' ) ) {
+			$actions[] = sprintf(
+				'<a href="%1$s">%2$s</a>',
+				esc_url( network_admin_url( "settings.php?page=plugin-check&plugin={$plugin_file}" ) ),
+				esc_html__( 'Check this plugin', 'plugin-check' )
+			);
+		} elseif ( current_user_can( 'activate_plugins' ) ) {
```

Comment:
See above, this could also support both WP Admin and Network Admin UIs.
```suggestion
		if ( is_network_admin() && current_user_can( 'manage_network_plugins' ) ) {
			$actions[] = sprintf(
				'<a href="%1$s">%2$s</a>',
				esc_url( network_admin_url( "settings.php?page=plugin-check&plugin={$plugin_file}" ) ),
				esc_html__( 'Check this plugin', 'plugin-check' )
			);
		} elseif ( ! is_network_admin() && current_user_can( 'activate_plugins' ) ) {
```